pom.xml

```diff
@@ -153,7 +153,7 @@
         <HikariCP.version>3.4.2</HikariCP.version>
         <jraft-core.version>1.3.12</jraft-core.version>
         <rpc-grpc-impl.version>${jraft-core.version}</rpc-grpc-impl.version>
-        <SnakeYaml.version>2.0</SnakeYaml.version>
+        <SnakeYaml.version>1.33</SnakeYaml.version>
```

Comment:
What are the reasons for using the lower version?

Reply:
I got a bug and found solution by the following link:
https://stackoverflow.com/questions/70154082/getting-java-lang-nosuchmethoderror-org-yaml-snakeyaml-yaml-init-while-runnin
bugs:
```bash
Caused by: java.lang.NoSuchMethodError: org.yaml.snakeyaml.constructor.SafeConstructor: method <init>()V not found
```